Stetskyi_Homework_13/ADOClassLibrary/ADOClassLibrary/Interfaces/IPerformer.cs

```diff
@@ -0,0 +1,23 @@
+﻿using ADOClassLibrary.Models;
+using System.Collections.Generic;
+
+
+namespace ADOClassLibrary.Interfaces
+{
+    public interface IPerformer
+    {
+        public List<Product> GetAllProducts();
+        public List<Order> GetAllOrders(int month = -1, string status = "", int year = -1, int productId = -1);
+        public void DeleteOrders(int month = -1, string status = "", int year = -1, int productId = -1);
+        public List<Order> GetAllOrders();
```

Comment:
@IhorStetskyi, I believe we should separate 'order' interface and 'product' interface.